getDbFunction.php

```diff
@@ -5,7 +5,7 @@
          * @return 
          */
         function getDb() : PDO {
-            $database = new PDO('mysql:host=db; dbname=MyCollection','root','password');
-            $database->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
-            return $database;
+        $database = new PDO('mysql:host=db; dbname=MyCollection','root','password');
+        $database->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+        return $database;
```

Comment:
Check your indents here as well, the contents of the function should be indented 1 level deeper, and the function declaration line shouldn't be indented at all as its not inside of anything else.